htdocs/includes/odtphp/odf.php

```diff
@@ -530,7 +530,7 @@ public function preOdfToOdf($value)
 	 * @throws OdfException
 	 * @return odf
 	 */
-	public function setImage($key, $value)
+	public function setImage($key, $value, float $ratio=1)
```

Comment:
You must add the parameter $ratio into the doc comment.
`@param float $ratio   Ratio for image`